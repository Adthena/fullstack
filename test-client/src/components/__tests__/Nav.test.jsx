import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom'

import Nav from '../Nav';

it('renders correctly', () => {
  const { container } = render(<Nav />, { wrapper: MemoryRouter });
  expect(container).toMatchSnapshot();
});
