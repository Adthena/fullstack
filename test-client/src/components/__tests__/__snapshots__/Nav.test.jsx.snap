// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <nav
    class="sc-bdfBwQ juAtS"
  >
    <a
      aria-current="page"
      class="sc-gsTCUz hVNGuK active"
      href="/"
    >
      Task One
    </a>
    <a
      class="sc-gsTCUz hVNGuK"
      href="/task-two"
    >
      Task Two
    </a>
  </nav>
</div>
`;
